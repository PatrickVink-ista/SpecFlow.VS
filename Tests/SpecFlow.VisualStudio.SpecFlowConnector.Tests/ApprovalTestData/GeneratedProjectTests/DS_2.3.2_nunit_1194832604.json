{
  "ConfigFile": "DeveroomSample.dll.config",
  "GeneratorOptions": {
    "Force": false,
    "_TargetFolder": null,
    "NewProjectFormat": false,
    "SpecFlowPackageVersion": "2.3.2",
    "UnitTestProvider": "NUnit",
    "FeatureFileCount": 3,
    "ScenarioPerFeatureFileCount": 5,
    "StepDefPerClassCount": 6,
    "StepDefinitionPerStepPercent": 70,
    "ScenarioOutlinePerScenarioPercent": 30,
    "FallbackNuGetPackageSource": null,
    "TargetFramework": "net48",
    "AddGeneratorPlugin": false,
    "AddRuntimePlugin": false,
    "PluginName": "unknown",
    "AddExternalBindingPackage": false,
    "ExternalBindingPackageName": "unknown",
    "AddUnicodeBinding": false,
    "AddAsyncStep": false,
    "IsBuilt": false,
    "PlatformTarget": null,
    "CreatedFor": null,
    "TargetFolder": "C:\\Users\\santa\\AppData\\Local\\Temp\\Deveroom\\DS_2.3.2_nunit_1194832604",
    "SpecFlowVersion": {
      "Major": 2,
      "Minor": 3,
      "Build": 2,
      "Revision": -1,
      "MajorRevision": -1,
      "MinorRevision": -1
    }
  }
}
