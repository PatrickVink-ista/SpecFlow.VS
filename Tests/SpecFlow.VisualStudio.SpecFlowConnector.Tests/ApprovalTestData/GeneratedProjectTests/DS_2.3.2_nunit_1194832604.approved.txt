{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "2.3.2",
    "SFProductVersion": "2.3.2",
    "TargetFramework": ".NETFramework,Version=v4.8"
  },
  "errorMessage": "<errorMessage>"
}